AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/services/allocate-hearings.service.ts

```diff
@@ -27,4 +27,11 @@ export class AllocateHearingsService {
             return new Observable<AllocationHearingsResponse[]>();
         }
     }
+
+    setAllocationToHearings(selectedHearings: string[], csoUserId: string): Observable<AllocationHearingsResponse[]> {
```

Comment:
setAllocationToHearings feels like there's too many words in that method. Could just be allocateHearings()

Reply:
sounds good! naming is sometime a mare